lib/events.js

```diff
@@ -154,10 +154,11 @@ EventEmitter.prototype.emit = function emit(type, ...args) {
 
   // If there is no 'error' event listener then throw.
   if (doError) {
-    let er;
+    let er, stringifiedEr;
     if (args.length > 0)
       er = args[0];
     if (er instanceof Error) {
+      stringifiedEr = er;
```

Comment:
This assignment is unused  because `er` is thrown before reaching any code below (https://github.com/nodejs/node/pull/25621/files#diff-71dcd327d0ca067b490b22d677f81966R174).